libevm/VM.cpp

```diff
@@ -115,6 +113,22 @@ void VM::adjustStack(unsigned _removed, unsigned _added)
 #endif
 }
 
+void VM::sstoreGas()
+{
+	if (m_ext->staticCall)
```

Comment:
Yeah, it's included as a preamble to lots of functions that way.  Somebody slipped them in while I wasn't looking ;)